forms/user.form.php

```diff
@@ -90,11 +89,8 @@
 		throw new InvalidArgumentException("Password cannot be empty and must be at least 8 characters");
 	}
 
-	// here we want to hash the pw and save the hash in the db
-	$hash = password_hash($password, PASSWORD_BCRYPT);
-
 	// creating a new user object
-	$user = new User($firstName, $lastName, $email, $hash, $tel, false);
+	$user = new User($firstName, $lastName, $email, $password, $tel, false);
```

Comment:
`\User`

Reply:
Added the backslash in commit 4af650ec6a7bfcf60b55a7760a82dfe96147776e